WebAPI/Controllers/CategoriesController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using WebNetSample.Business.Abstract;
+
+namespace WebNetSample.WebAPI.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class CategoriesController : ControllerBase
+{
+    private readonly ICategoryService _categoryService;
+
+    public CategoriesController(ICategoryService categoryService)
+    {
+        _categoryService = categoryService;
+    }
+
+    [HttpGet("getall")]
+    public async Task<IActionResult> GetAll()
```

Comment:
For Web API controllers, please, use [`ActionResult<T>`](https://learn.microsoft.com/en-us/aspnet/core/web-api/action-return-types?view=aspnetcore-7.0#actionresultt-type)

(Same case for products)